samples/CommunityToolkit.Maui.Sample/Platforms/Android/MainActivity.cs

```diff
@@ -1,10 +1,17 @@
 ﻿using Android.App;
 using Android.Content.PM;
+using Android.OS;
 using Microsoft.Maui;
 
 namespace CommunityToolkit.Maui.Sample;
 
 [Activity(Theme = "@style/Maui.SplashTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation | ConfigChanges.UiMode | ConfigChanges.ScreenLayout | ConfigChanges.SmallestScreenSize)]
 public class MainActivity : MauiAppCompatActivity
 {
+	// https://github.com/dotnet/maui/pull/3345
```

Comment:
Appreciate the update, but let's remove the comment/link from the source code

Reply:
let's keep it while the issue exists. we can remove the whole method later